lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -120,6 +121,7 @@ def initialize(namespace:, context:, current_sha:, template_dir:, logger:, kubec
         logger: @logger,
         bindings: bindings,
       )
+      @selector = selector
     end
 
     def run(*args)
```

Comment:
I can't comment directly on the relevant line, but I think we should log something like `Using resource selector {}`

![image](https://user-images.githubusercontent.com/4789493/54062961-6a609780-41d7-11e9-938b-961ea22dd040.png)

Right now there's no way to tell from the logs that the deploy is only managing a subsection of the namespace. The logging assertions in the tests should be augmented to check for this too, assuming we add it.